server.js

```diff
@@ -15,7 +15,10 @@ server.use(express.urlencoded({ extended: false }));
 
 async function sendEmail(formData) {
```

Comment:
Lets destructure this as

`
async function sendEmail({firstName, lastName, email, orderInfo})
`

you can use them directly in the code below

Reply:
done